src/app/modules/core/components/footer/footer.component.scss

```diff
@@ -1,158 +1,67 @@
 @import "~assets/styles/variables";
 
-:host ::ng-deep {
-
-  .navbar-nav {
-    flex-direction: row;
-    justify-content: center;
-
-    .nav-link {
-      padding: 15px;
-      color: $secondary-color;
-
-      &:hover {
-        color: $main-color-darker;
-      }
-    }
-
-    .active {
-      color: $main-color-darker;
-    }
-  }
-
-  .forum {
-    display: block !important;
-  }
-}
-
 .footer {
-  font-size: 12px;
-  margin-top: 50px;
-  padding-bottom: 10px;
-  width: 100%;
-  background-repeat: no-repeat;
-  background-size: 100% 4px;
-  background-image: linear-gradient(120deg, $main-color-lighter 0%, $main-color 33%, $main-color-darker 100%);
+  height: 392px;
+  width: $container-width;
+  background-color: $background-main-color-darker;
 
-  &__row {
-    display: flex;
-    align-items: center;
-    padding: 10px 0 0;
-  }
-
-  .container {
-    display: flex;
+  .footer__links {
+    height: 71.5px;
     flex-direction: row;
-    justify-content: center;
-    align-items: center;
+    list-style: none;
+    gap: 40px;
+    font-size: $font-size-small;
+    color: $font-color-primary;
+    border-top: .05px solid $font-color-dark;
 
-    .social {
-      display: flex;
-      flex-direction: row;
-      align-items: center;
-      margin: 0;
-      padding: 0;
-      list-style: none;
-    }
+    @include flex-center;
   }
 
   .copy {
-    display: flex;
-    justify-content: center;
-    align-items: center;
     opacity: .5;
-    font-size: 14px;
-    color: $secondary-color;
-
-    .text::after {
-      content: " ";
-      white-space: pre;
-    }
 
     a {
-      color: $main-color-darker;
+      color: $font-color-primary;
+      text-decoration: underline;
+      text-underline-offset: 5px;
     }
   }
 
-  &__icon {
-    transition: .3s ease;
+  & svg {
+    background-color: $background-color-white;
   }
 
-  &__link {
-    display: flex;
-    margin-left: 10px;
-
-    &_ucf {
-      @media (max-width: 476px) {
-
-        .footer__icon {
-          width: 50px;
-          height: 40px;
-        }
-      }
-
-      &:hover .footer__icon {
-        fill: #ffcc2d;
-      }
-    }
-
-    &:hover {
-
-      .footer__icon {
-        fill-opacity: .8;
-      }
-    }
+  &__icon {
+    transition: .3s ease;
   }
+}
 
+.ucf-text {
+  color: $font-color-white;
+}
 
-  @media (max-width: 991.98px) {
-
-    .footer-menu {
-      display: none;
-    }
-
-    .social {
-      margin-left: auto;
-    }
-
-    .copy {
-      padding-top: 5px;
-    }
-
-    &__row {
-      justify-content: center;
-    }
-
-    &__link {
-      margin-top: 14px;
-    }
-  }
-
-  .copy {
-    @media (max-width: 476px) {
-      display: flex;
-      flex-direction: column;
-    }
-  }
+.footer__link {
+  margin-left: 7px;
 }
 
 .scrollToTop {
-  z-index: 999;
-  font-size: 0;
+  z-index: 20;
   position: absolute;
-  bottom: 67px;
+  margin-top: -500px;
```

Comment:
??

Reply:
Changed the position of the "scroll to top" button.